src/components/donation-flow/DonationFlowPage.tsx

```diff
@@ -1,137 +1,104 @@
 import Link from 'next/link'
 import Image from 'next/image'
 import { styled } from '@mui/material/styles'
-import { Box, Grid, Typography, useMediaQuery } from '@mui/material'
+import { Box, Unstable_Grid2 as Grid2, Typography, useMediaQuery } from '@mui/material'
```

Comment:
@dimitur2204 Thanks for using `Grid2` - it structures the content in much better way :rocket: